WcaOnRails/app/models/concerns/registrations_notifications.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'active_support/concern'
+
+module RegistrationsNotifications
+  extend ActiveSupport::Concern
+
+  included do
+    # NOTE: it turns out user can be nil in tests.
```

Comment:
Wouldn't it make more sense to fix the tests then, instead of being "sloppy" with a nil-safe operator here? How much more additional work would changing the tests entail?

Reply:
That's a very good point!

The failing tests in itself are fairly weird because they intentionally introduce an invalid record to make sure save a competition cleans it, but I don't see how one would end up in the db in the first place :s

Anyway I've adjusted the concern to assume user is not nil (which is reasonable in normal website usage), and updated the two failing tests to skip callbacks.

Reply:
Cool, thanks! 😄